pkg/ccl/logictestccl/testdata/logic_test/as_of

```diff
@@ -29,6 +29,9 @@ BEGIN AS OF SYSTEM TIME follower_read_timestamp(); SELECT * FROM t
 statement ok
 ROLLBACK
 
+statement error pgcode 0A000 cannot specify AS OF SYSTEM TIME with different timestamps
+SELECT * from t AS OF SYSTEM TIME '-1μs'; SELECT * from t AS OF SYSTEM TIME '-2μs'
```

Comment:
cc @otan to make this fail you need to set enable_implicit_transaction_for_batch_statements to true